package.json

```diff
@@ -34,6 +34,6 @@
     "xml": "^1.0.0"
   },
   "peerDependencies": {
-    "mocha": "^2.2.5"
+    "mocha": "^3 || ^2.2.5"
```

Comment:
@michaelleeallen @clayreimann Okay so… should I change it or not? You're both saying different things